docs/src/pages/components/tabs/UnstyledTabsCustomized.js

```diff
@@ -38,10 +38,10 @@ const Tab = styled(TabUnstyled)`
     background-color: ${blue[400]};
   }
 
-  &.${buttonUnstyledClasses.focusVisible} {
```

Comment:
Don't we encourage the community to use `focusVisible`? cc @michaldudak

Reply:
I changed to only `&:focus` because using the `focusVisible` class wasn't working 😕  The styles weren't being applied to the component when focused. Not sure if it's really a problem or just me not knowing how to work on it.